modal/mount.py

```diff
@@ -24,10 +25,18 @@ def client_mount_name():
     return f"modal-client-mount-{__version__}"
 
 
+@dataclasses.dataclass
+class _MountEntry:
+    local_dir: Optional[Path]
+    local_file: Optional[Path]
+    path_within_mount: Optional[Path]
```

Comment:
Maybe :thinking: I still think the local path should have "local" in it. Maybe `mount_path`? Although that could be confused for the path that something is mounted into in the final container, which it isn't necessarily (since the mount can get a different root at the function specification level as well)